v2/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "tupelo-wasm",
+  "version": "0.0.1",
+  "description": "a test interaction of tupelo wasm",
+  "main": "app.js",
+  "scripts": {
+    "build": "tsc -p .",
+    "test": "mocha --exit -r ts-node/register src/*.spec.ts",
+    "build:wasm": "cd ./tupelo-go-sdk/wasm && GOOS=js GOARCH=wasm go build -o ../../main.wasm"
+  },
+  "author": "",
+  "license": "rights reserved.",
+  "dependencies": {
+    "@types/protobufjs": "^6.0.0",
+    "cids": "^0.7.1",
+    "debug": "^4.1.1",
+    "events": "^3.0.0",
+    "google-protobuf": "^3.9.0",
+    "interface-datastore": "^0.7.0",
+    "ipfs-block-service": "^0.16.0",
+    "ipfs-repo": "^0.27.0",
+    "ipld": "quorumcontrol/js-ipld#upgrade/block-service-async",
```

Comment:
IPLD currently expects a callback-based ipfs-block-service but it has moved on to an async/await as of version 0.16.0